docs/source/resources/scyllaclusters/multidc/multidc.md

```diff
@@ -110,7 +110,7 @@ metadata:
   name: scylla-cluster
   namespace: scylla
 spec:
-  agentVersion: 3.5.0
+  agentVersion: {{agentVersion}}
```

Comment:
I am not sure about using agentVersion.

This variable is set across all Operator versions in `conf.py`, while the manager version is pinned only in that specific version. This will cause drift that I am not sure is acceptable.

Suggestion: keep hardcoding until we have a better solution for variable versioning 😢

Reply:
Should we remove substitutions everywhere then? It's used in some docs and hardcoded in others, whatever we do it's probably best to make it consistent.

Reply:
Yup, to avoid future confusion I would remove the variable from myst_substitutions altogether.

Reply:
@mflendrich https://github.com/scylladb/scylla-operator/compare/b0cf53ad5ee7a574f64ce8d6752537d1b03e5c06..73b7ac6bcc8b8d9834096292dcd5c8901f1a5b49